### PR TITLE
Fix seeders/leechers and info beeing removed from ttn

### PR DESCRIPTION
**Pull request by duramato**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read [contribution guide](https://github.com/PyMedusa/SickRage/blob/master/.github/CONTRIBUTING.md)

Fixes: https://github.com/pymedusa/SickRage/issues/475#issuecomment-242038736

### CODE REVIEW

File: sickbeard/providers/torrent/html/transmitthenet.py [hunk 1]
```diff
@@ -148,13 +148,13 @@ def parse(self, data, mode):
                     download_url = urljoin(self.url, download_item['href'])
 
                     temp_anchor = row.find('a', {'data-src': True})
-                    title = temp_anchor['data-src'].rsplit('.', 1)[0]
```
Comment (duramato) [inline sickbeard/providers/torrent/html/transmitthenet.py]: This was causing for stuff from beeing removed from release names, not all are scene.

**Comment by fernandog** [comment]
@medariox ok?

**Comment by medariox** [comment]
Don't have access to the provider atm. The changes look good. If @duramato tested manual and daily search, I think it should be fine.

**Comment by fernandog** [comment]
Approved

**Pull request merged**